network-api/networkapi/mozfest/models.py

```diff
@@ -98,6 +98,8 @@ def get_context(self, request, bypass_menu_buildstep=False):
 
 
 class MozfestHomepage(MozfestPrimaryPage):
+    banner_type = "video"
```

Comment:
Let's add some comments here that explains why this field exists, because it's sort of a sneaky "not django, not wagtail" field. Probably a little more than a one liner, using python's block-of-string comment syntax:

```python
class MozfestHomepage(MozfestPrimaryPage):
    """
    ...description of this class and the banner_type value in particular
    """
    banner_type = "...."
```